### DISCUSSION TITLE
Process when describing MIMIC-III data in publicly available documents

### DISCUSSION THREAD

**Discussion by philipobrien** [MIT-LCP/mimic-code#1720] [MIMIC-General]
Hi, I'm involved with an EU funded project which will have a number of publicly available deliverables. One of those is around the area of explainable AI. I have been working with the MIMIC-III dataset (having gone through the process on physionet) to deepen my knowledge of XAI techniques, e.g. building models against the data and using various XAI tools to interpret these models. 
When discussing the use of MIMIC in this regard, within the written deliverables, do I need to avoid screenshots/descriptions of actual data values e.g. even something small like a screenshot of the first few rows of a dataframe corresponding to one of the MIMIC-III datasets.
I know of course to reference the work/authors appropriately, but I just want to be sure I am doing things right.

**Comment by tompollard** [comment]
Thanks for checking @philipobrien 

> do I need to avoid screenshots/descriptions of actual data values e.g. even something small like a screenshot of the first few rows of a dataframe corresponding to one of the MIMIC-III datasets.

Yes, that's right. Even screenshots of the data would be considered sharing, which is prohibited by the Data Use Agreement that you signed. 

There is an open access dataset at https://physionet.org/content/mimiciii-demo/1.4/ that you could use instead. If resharing parts of this dataset, please be explicit that it is from the demo (and not the full, protected MIMIC dataset).

**Reply by philipobrien** [reply]
Thanks for the quick response! Something I am very interested in is the use of [SHAP](https://shap.readthedocs.io/en/latest/index.html) for interpreting model predictions. Is it OK to include SHAP plots in the document, once they have no personally identifiable information e.g. take a simple case of predicting mortality using input variables such as age, length of stay, admission type.....the SHAP plot would list these variables along with their importance in predicting the model output, but obviously nothing like subject_id of the patient.

**Reply by philipobrien** [reply]
Hi again, just following up on this?

**Reply by tompollard** [reply]
Apologies @philipobrien

> e.g. take a simple case of predicting mortality using input variables such as age, length of stay, admission type.....the SHAP plot would list these variables along with their importance in predicting the model output

This would be acceptable, thanks for checking. Our primary concern is risk of revealing sensitive data, and we don't intend to limit your ability to report on study results.

**Reply by philipobrien** [reply]
I completely understand. Thanks for clarifying.
